packages/ui/src/theme/css/styles.scss

```diff
@@ -26,6 +26,7 @@
 
 // Composed components
 @import './component/accordion.scss';
+@import './component/aiconversation.scss';
```

Comment:
Would it make sense to put the Conversation stylesheets directly in the ui-react-ai package?

Reply:
Ah I was just copying our existing patterns, would you envision the customer doing something like below instead? Not opposed to it but not sure what the implications are

```
import '@aws-amplify/ui-react-ai/styles.css';
```

Reply:
Maybe we should discuss with @dbanksdesign, my thinking is that since the base classes applied to the `element` are only layout related they could live independently from the our standard stylesheets